### DISCUSSION TITLE
tailwindcss有可能回来吗

### DISCUSSION THREAD

**Discussion by leessmin** [vbenjs/vue-vben-admin#3572] [General]
我习惯用tailwindcss了，用uno有点不太习惯，未来有没有选择的余地？将tailwind和uno做成二选一？

**Comment by kkfive** [comment]
> 我习惯用tailwindcss了，用uno有点不太习惯，未来有没有选择的余地？将tailwind和uno做成二选一？

unocss 使用tailwindcss预设 会有很大的差别么

**Reply by leessmin** [reply]
又些许区别，比如没法用tailwind catppuccin

**Comment by likui628** [comment]
> 我习惯用tailwindcss了，用uno有点不太习惯，未来有没有选择的余地？将tailwind和uno做成二选一？

v2版本应该是不会了，v5是tailwind
